fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/NotificationService.java

```diff
@@ -101,6 +105,14 @@ static void addNotificationToPassedMap(String targetFile, Notification notificat
      * NOTIFICATION_DIRECTORY_PATH.
      */
     public void recordNotifications(MavenProject project) {
```

Comment:
Q/S: Where is this being used? I wouldn't think we'd ever want to call this method _without_ supplying the suppression configuration.